### ISSUE TITLE
Off-by-one in TrailingSemicolonAfterPropertyValue linter

### ISSUE THREAD

**Issue by asottile** [brigade/scss-lint#163]
``` css
.foo { border: 0 }
```

```
SCSSLint::Linter::TrailingSemicolonAfterPropertyValue raised unexpected error linting file test.css: 'Line must be more than 0, passed 0'
.../repoNxk7pR/rbenv/gems/gems/scss-lint-0.23.1/lib/scss_lint/location.rb:12:in `initialize'
.../repoNxk7pR/rbenv/gems/gems/scss-lint-0.23.1/lib/scss_lint/linter.rb:140:in `new'
.../repoNxk7pR/rbenv/gems/gems/scss-lint-0.23.1/lib/scss_lint/linter.rb:140:in `extract_location'
.../repoNxk7pR/rbenv/gems/gems/scss-lint-0.23.1/lib/scss_lint/linter.rb:30:in `add_lint'
.../repoNxk7pR/rbenv/gems/gems/scss-lint-0.23.1/lib/scss_lint/linter/trailing_semicolon_after_property_value.rb:18:in `visit_prop'
.../repoNxk7pR/rbenv/gems/gems/sass-3.3.7/lib/sass/tree/visitors/base.rb:36:in `visit'
.../repoNxk7pR/rbenv/gems/gems/scss-lint-0.23.1/lib/scss_lint/linter.rb:115:in `visit'
.../repoNxk7pR/rbenv/gems/gems/scss-lint-0.23.1/lib/scss_lint/linter.rb:125:in `block in visit_children'
.../repoNxk7pR/rbenv/gems/gems/scss-lint-0.23.1/lib/scss_lint/linter.rb:123:in `each'
.../repoNxk7pR/rbenv/gems/gems/scss-lint-0.23.1/lib/scss_lint/linter.rb:123:in `visit_children'
.../repoNxk7pR/rbenv/gems/gems/sass-3.3.7/lib/sass/tree/visitors/base.rb:38:in `visit'
.../repoNxk7pR/rbenv/gems/gems/scss-lint-0.23.1/lib/scss_lint/linter.rb:115:in `visit'
.../repoNxk7pR/rbenv/gems/gems/scss-lint-0.23.1/lib/scss_lint/linter.rb:125:in `block in visit_children'
.../repoNxk7pR/rbenv/gems/gems/scss-lint-0.23.1/lib/scss_lint/linter.rb:123:in `each'
.../repoNxk7pR/rbenv/gems/gems/scss-lint-0.23.1/lib/scss_lint/linter.rb:123:in `visit_children'
.../repoNxk7pR/rbenv/gems/gems/sass-3.3.7/lib/sass/tree/visitors/base.rb:38:in `visit'
.../repoNxk7pR/rbenv/gems/gems/scss-lint-0.23.1/lib/scss_lint/linter.rb:115:in `visit'
.../repoNxk7pR/rbenv/gems/gems/scss-lint-0.23.1/lib/scss_lint/linter.rb:18:in `run'
.../repoNxk7pR/rbenv/gems/gems/scss-lint-0.23.1/lib/scss_lint/runner.rb:43:in `block in find_lints'
.../repoNxk7pR/rbenv/gems/gems/scss-lint-0.23.1/lib/scss_lint/runner.rb:38:in `each'
.../repoNxk7pR/rbenv/gems/gems/scss-lint-0.23.1/lib/scss_lint/runner.rb:38:in `find_lints'
.../repoNxk7pR/rbenv/gems/gems/scss-lint-0.23.1/lib/scss_lint/runner.rb:22:in `block in run'
.../rbenv/gems/gems/scss-lint-0.23.1/lib/scss_lint/runner.rb:21:in `each'
.../epoNxk7pR/rbenv/gems/gems/scss-lint-0.23.1/lib/scss_lint/runner.rb:21:in `run'
.../epoNxk7pR/rbenv/gems/gems/scss-lint-0.23.1/lib/scss_lint/cli.rb:94:in `run'
.../rbenv/gems/gems/scss-lint-0.23.1/bin/scss-lint:7:in `block in <top (required)>'
.../repoNxk7pR/rbenv/gems/gems/scss-lint-0.23.1/bin/scss-lint:5:in `tap'
.../epoNxk7pR/rbenv/gems/gems/scss-lint-0.23.1/bin/scss-lint:5:in `<top (required)>'
.../epoNxk7pR/rbenv/gems/bin/scss-lint:23:in `load'
.../repoNxk7pR/rbenv/gems/bin/scss-lint:23:in `<main>'
Report this bug at https://github.com/causes/scss-lint/issues
```

Related:

``` scss
 // this is line one
.foo { border: 0 }
```

```
$ scss-lint test.css
test.css:1 [W] Property declaration should not have a space before the terminating semicolon
```

Reports on line 1 instead of line 2
